priv/templates/extended_bin

```diff
@@ -416,8 +428,10 @@ case "$1" in
         echo "Exec: $@" -- ${1+$ARGS}
         echo "Root: $ROOTDIR"
 
-        # Start the VM
+        # Start the VM, invoke pre/post hooks
+        [ "$PRE_START_HOOK" ] && . "$PRE_START_HOOK"
         exec "$@" -- ${1+$ARGS}
+        [ "$POST_START_HOOK" ] && . "$POST_START_HOOK"
```

Comment:
i've since updated the PR, it now sends the execs to background and waits on them after the post hook